templates/member/index.html.twig

```diff
@@ -48,7 +48,8 @@
                                 <li class="list-group-item m-2 fw-bold d-flex flex-row justify-content-between">
                                     {{ member.cardNumber }}-{{ member.name }}
                                     <div class="d-flex flex-row">
-                                        {% if loop.first %}
+                                        {% if loop.first == true and loop.last == true %}
```

Comment:
un peu compliqué ton truc. Fais l'inverse plutot :

if not loop.first
=> j'affiche la flèche haute
if not loop.last
=> j'affiche la flèche basse